data/food/export_agb/export_builder.py

```diff
@@ -53,6 +53,13 @@ def search(name):
     assert len(results) >= 1, f"'{name}' was not found in Brightway"
     return results[0]
 
+def find_id(activity):
+    # if this is a complex ingredient, the id is the one constructed by ecobalyse
+    if "ratio" in activity.keys():
```

Comment:
So if we have a ratio, it's the sign of a complex ingredient. So why do we also need a `complex` boolean? Isn't it redundant?